src/test/scala/mesosphere/marathon/integration/setup/ProcessKeeper.scala

```diff
@@ -33,6 +33,14 @@ object ProcessKeeper {
 
   private[this] val ENV_MESOS_WORK_DIR: String = "MESOS_WORK_DIR"
 
+  case class MesosConfig(
+    port: Int = 5050,
```

Comment:
Do not hardcode ports, especially not one I'm probably already using mesos on. (Use PortAllocator)